homeassistant/components/slide_local/entity.py

```diff
@@ -0,0 +1,34 @@
+"""Entities for slide_local integration."""
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import SlideCoordinator
+
+
+class SlideEntity(CoordinatorEntity[SlideCoordinator]):
+    """Base class of a Slide local API cover."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: SlideCoordinator,
+    ) -> None:
+        """Initialize the Slide device."""
+        super().__init__(coordinator)
+
+        self._attr_device_info = DeviceInfo(
+            manufacturer="Innovation in Motion",
+            identifiers={(DOMAIN, coordinator.data["mac"])},
```

Comment:
Let's use connections instead

Reply:
Done. What is the difference/benefit? I also saw, that sometimes both is uses (e.g. wemo)

Reply:
they are both just used for identifying, but only identifiers can be used with `via_device`

Reply:
Which would rather speak for using identifiers above connections...
Anyway, have changed it now

Reply:
But in this case we don't have to use identifiers since it's 1 config entry - 1 device so there's no need for via_device